src/OpenTelemetry.Instrumentation.Runtime/CHANGELOG.md

```diff
@@ -2,13 +2,17 @@
 
 ## Unreleased
 
+## 0.3.0-alpha.1
+
 * Update a few metrics to be in sync of doc updates for `gc.heap`,
   `gc.fragmentation.ratio`, `time.in.jit`, `process.cpu.count` and `assembly.count`
   ([#430](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/430))
 * Remove Process related metrics from .NET Runtime metrics
   ([#446](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/446))
 * Add `exception.count` in Runtime metrics
   ([#431](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/431))
+* Finalize GC instruments
```

Comment:
I think we should group all these changes into a single summary.

Reply:
All the five bullet points into one "major refactoring"?

Reply:
Personally I think the changelog between these versions don't matter much until we have our first beta or stable version.

Reply:
My main consideration - it's very hard for the user to go through the changelog and get the key message, unless they parse it sequentially (e.g. one would read "process.cpu.count was updated", later notice "process related metrics are actually removed").

If a developer want to know the change history, I think git log / commit history is a perfect place. CHANGELOG to me is a communication channel to the user regarding "what should they expect".

Reply:
I've updated it.